cfg/RoombaEstimator.cfg

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+PACKAGE = "iarc7_vision"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+import math
+
+gen = ParameterGenerator()
+
+gen.add('hsv_slice_h_green_min', int_t, 0, '', 0, 0, 180)
+gen.add('hsv_slice_h_green_max', int_t, 0, '', 0, 0, 180)
+gen.add('hsv_slice_h_red1_min',  int_t, 0, '', 0, 0, 180)
+gen.add('hsv_slice_h_red1_max',  int_t, 0, '', 0, 0, 180)
```

Comment:
Why two red slices?

Reply:
Red is near 0 hue, so there's some red just above 0 and some just below 180.